main.go

```diff
@@ -31,6 +31,398 @@ const bwSuccessPrefix = "✓"
 const bwFailPrefix = "𐄂"
 const bwErrorPrefix = "!"
 
+/**
+ * Call function and, when it is possible display spinner with some progress message
+ */
```

Comment:
```suggestion
// showProgress calls function and, when it is possible display spinner with
// some progress message.
```

There's a [godoc convention](https://go.dev/blog/godoc) that stipulates that documentation begins with the name of the variable, type, or function/method its documenting. I don't think it's a strict requirement, and `go doc` should still parse the documentation, but it's a common convention.

I've also not seen much use of the `/* */` comment block syntax, though it's certainly supported. Most comment documentation tends to just use line after line of double-slash comments.

Here's what `go doc` outputs for `/* */` comments:

```
[link@thelio rhc]$ go doc ShowProgress
func ShowProgress(progressMessage string, isColorful bool, function func() error) error
    *
      - Call function and, when it is possible display spinner with some
        progress message
```

And here's what it does with double-slash syntax.
```
[link@thelio rhc]$ go doc ShowProgress
func ShowProgress(progressMessage string, isColorful bool, function func() error) error
    ShowProgress calls function and, when it is possible display spinner with
    some progress message.
```

NB: `go doc` only processes exported symbols, so for the sake of the example above, I temporarily changed `showProgress` to `ShowProgress` to demonstrate the doc comment parsing. It's still good practice to use the same convention when documenting unexported symbols.

Reply:
OK, no problem, I can use this pattern.